.travis.yml

```diff
@@ -44,7 +44,7 @@ matrix:
         - TESTMODE=fast
         - COVERAGE=
         - NPY_RELAXED_STRIDES_CHECKING=1
-        - NUMPYSPEC="git+git://github.com/numpy/numpy.git@v1.9.1"
+        - NUMPYSPEC="--upgrade git+git://github.com/numpy/numpy.git@v1.9.1"
```

Comment:
@ev-br Would you suggest `--force-reinstall` and/or `--ignore-installed` instead?  Regarding turning the diagnostic print into an actual assertion, do you mean something involving bash or do you mean adding a scipy unit test?  Also feel free to make a replacement PR; I'm just messing around with this one and I'd be OK to close it.